src/components/WebexMemberRoster/WebexMemberRoster.jsx

```diff
@@ -24,11 +24,31 @@ import useMemberships from '../hooks/useMemberships';
  */
 export default function WebexMemberRoster({destinationID, destinationType}) {
   const memberships = useMemberships(destinationID, destinationType);
```

Comment:
let's call this members instead of memberships, they are actually members. We might also rename useMemberships to useMembers in the future.

Reply:
@cipak, make sure that all changes to the interface are first done on the [interface repo](https://github.com/webex/component-adapter-interfaces)

Reply:
ok, we'll create a PR for adding the `isInMeeting` field to the `member` interface. Other than that, the `useMemberships` hook  already returns a list of members, I was just trying to reflect that in this variable's name.

Reply:
I mistook this line as a change to the adapter's method. In this case, you're right, we should update this hook as `useMembers` to reflect its return values 👍🏼

Please make this change in a separate commit `refactor(hooks): ...`